tools/deploy_assisted_installer.py

```diff
@@ -63,10 +63,10 @@ def main():
             service_container["volumeMounts"].append({'name': 'iso', 'mountPath': "/data"})
             spec["volumes"].append({'name': 'iso', 'emptyDir': {}})
             spec["initContainers"] = [{
-                "name": "assisted-iso-create",
-                "image": "quay.io/ocpmetal/assisted-iso-create:latest",
+                "name": "rhcos-download",
+                "image": "quay.io/app-sre/centos:8",
                 "command": ["bash", "-c"],
-                "args": ["cp -r /data/* /iso-data"],
+                "args": ["until curl https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.6/latest/rhcos-live.x86_64.iso --retry 5 -o /iso-data/livecd.iso; do sleep 1; done"],
```

Comment:
Doesn't this need to be dependent on the `OPENSHIFT_VERSION`?
Like `https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/{OPENSHIFT_VERSION}/latest/rhcos-live.x86_64.iso`

Reply:
Since we are falling back to this image if one isn't provided to the assisted-service at runtime, I elected to just remove the initContainer. That prevents us from having to maintain it in more than the 1) Makefile and 2) s3wrapper. WDYT?